integration_tests/snaps/local-source-subfolders/packaging/snap-package/yes-really-deep/snapcraft.yaml

```diff
@@ -0,0 +1,12 @@
+name: lets-dump
+version: '0.1'
+summary: Let's dump parent folders
+description: Dumping up-folder
+
+grade: devel
+confinement: strict
+
+parts:
+  dump-part:
+    plugin: dump
+    source: ../../..
```

Comment:
this doesn't feel really really deep, one more level will though :blush: 

:smile: